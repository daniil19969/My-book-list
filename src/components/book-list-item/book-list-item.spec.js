import React from "react";
import BookListItem from "./book-list-item";
import DummyBookstoreService from '../../services/dummy-bookstore-service'
describe("BookListItem component", () => {
  it("should render component BookListItem", () => {
    const wrapper = shallow(<BookListItem book={DummyBookstoreService}/>);
    expect(wrapper).toMatchSnapshot();
  });
});