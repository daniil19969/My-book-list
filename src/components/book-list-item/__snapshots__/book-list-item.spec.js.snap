// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookListItem component should render component BookListItem 1`] = `
<Fragment>
  <td>
    <img
      alt="cover"
      className="smallImage"
    />
  </td>
  <td />
  <td />
  <td />
</Fragment>
`;
